CONTRIBUTING.md

```diff
@@ -158,16 +127,7 @@ Once your merge request is approved it will enter the merge queue
 [darach]: https://github.com/darach
 
 Speaking of tests, Rust has a comprehensive test suite. More information about
-it can be found [here][https://github.com/wayfair-incubator/tremor-runtime/blob/master/docs/development/testing.md].
-
-### External Dependencies
-
-Currently building the Tremor project will also build the following external projects:
-
-* [clippy](https://github.com/rust-lang/rust-clippy)
-* [rustfmt](https://github.com/rust-lang/rustfmt)
-
-Breakage is not allowed in released branches and must be addressed before a PR is merged.
+it can be found [here][https://github.com/wayfair-incubator/tremor-vscode/blob/master/docs/development/testing.md].
```

Comment:
should say wayfair-tremor

```suggestion
it can be found [here][https://github.com/wayfair-tremor/tremor-vscode/blob/master/docs/development/testing.md].
```